lib/open_food_network/users_and_enterprises_report.rb

```diff
@@ -63,14 +65,25 @@ def users_and_enterprises
     end
 
     def sort(results)
+      pp results
       results.sort do |a,b|
-        [ DateTime.parse(b["created_at"]), a["name"], b["relationship_type"], a["user_email"] ] <=>
-        [ DateTime.parse(a["created_at"]), b["name"], a["relationship_type"], b["user_email"] ]
+        if a["created_at"].nil? || b["created_at"].nil?
+          [ (a["created_at"].nil? ? 0 : 1), a["name"], b["relationship_type"], a["user_email"] ] <=>
+          [ (b["created_at"].nil? ? 0 : 1), b["name"], a["relationship_type"], b["user_email"] ]
+        else
+          [ DateTime.parse(b["created_at"]), a["name"], b["relationship_type"], a["user_email"] ] <=>
+          [ DateTime.parse(a["created_at"]), b["name"], a["relationship_type"], b["user_email"] ]
+        end
       end
     end
 
     def to_bool(value)
       ActiveRecord::ConnectionAdapters::Column.value_to_boolean(value)
     end
+
+    def to_local_datetime(string)
+      return "" if string.nil?
+      string.to_datetime.in_time_zone.strftime "%Y-%m-%d %H:%M"
+    end
```

Comment:
That was what the original method looked like in `master`, I was just putting it back in. I'll change it now...